src/main/java/edu/cornell/kfs/fp/batch/service/impl/AdvanceDepositServiceImpl.java

```diff
@@ -186,16 +192,31 @@ public boolean routeAdvanceDepositDocuments() {
                     LOG.info("Routing advance deposit document # " + documentId + ".");
                 }
                 documentService.routeDocument(advanceDocument, "document routed by achIncome batch job", null);
-            } catch (WorkflowException e) {
-                LOG.error("Error routing document # " + documentId + " " + e.getMessage());
-                throw new RuntimeException(e.getMessage(), e);
+            }
```

Comment:
Good question. Seems though that if there is a problem retrieving the docs that is kind of a legitimate failure and throwing the exception seems fine.